Realm/RLMRealm.mm

```diff
@@ -477,6 +478,9 @@ + (instancetype)realmWithConfiguration:(RLMRealmConfiguration *)configuration
     }
 #endif
 
+    // Run Analytics and Update checker, this will be run only the first any realm open
+    [self runFirstCheckForConfiguration:configuration schema:realm.schema];
```

Comment:
This is moving the call to set_default_logger() to too late in the opening process.

Reply:
I moved to `initialize`, but this will be replaced by the implementation on the Core's Logging PR when we are actually setting a default logger and a level threshold.